docs/backup-restore.md

```diff
@@ -1,32 +1,104 @@
-That means that you need to create a consistent snapshot by dumping all tables in a single transaction.
-For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
-Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
+You must create a Cosmos DB account with a Continuous backup policy enabled to create point-in-time restore (PITR). Backups are created continuously after enabling this.
+For creating a transactionally-consistent restore point, you can refer to [the basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup).
+
+**DynamoDB**
+
+You can enable the point-in-time recovery (PITR) feature for tables in DynamoDB. Point-in-time recovery(PITR) creates a continuous backup for a table.
+For selecting a restore point, you can refer to [the basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup).
+You can enable auto-scaling if you want. Auto-scaling helps to maintain throughput capacity based on workload. It helps in improving efficiency and reducing costs.
+Scalar DB Schema Loader enables PITR and auto-scaling by default.
+
+## Restore Backup
+
+### JDBC databases
+
+You can restore the backup with your favorite way for JDBC databases.
+You can use `mysql` command in MySQL and `psql` command in PostgreSQL to achieve that.
+If you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL,
+you can restore to any point within the backup retention period with the automated backup feature.
+
+### Cassandra
+
+To restore the backup, you should stop the Cassandra and clean the `data`, `commitlogs`, and `hints` directories then restore the backup copies of all the nodes of the cluster, and start the cluster.
+
+To avoid mistakes when doing restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+You can restore to any point using `CLUSTER-ID` and `SNAPSHOT-ID`.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### Cosmos DB
+
+To restore a backup, please follow the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and change the default consistency to `STRONG` after restoring the backup.
+It is recommended to use the midtime of paused duration as a restore point as we explained earlier.
+
+### DynamoDB
+
+You can restore the table to a point in time using the DynamoDB console or the AWS Command Line Interface (AWS CLI). The point-in-time recovery process restores to a new table.
+The tables can only be restored one by one.
```

Comment:
`The tables`, What `tables`?

You mean this? or specifying some particular tables?
```suggestion
Tables can only be restored one by one.
```

Reply:
Updated the sentence.